tests/test_logic.py

```diff
@@ -279,10 +280,10 @@ def test_minimize():
     C = Clauses(15)
     C.Require(C.ExactlyOne, range(1, 6))
     sol = C.sat()
-    C.unsat = True
+    C._clauses.unsat = True
```

Comment:
I did not expose a setter for the `Clauses.m` property since there should not be any need to set from outside it in actual code. Hence we need to use `._clauses.m` here.